docs/source/install/notebook-classic.rst

```diff
@@ -13,9 +13,8 @@ kernel.
 Prerequisite: Python
 --------------------
 
-While Jupyter runs code in many programming languages, **Python** is
-a requirement (Python 3.3 or greater, or Python 2.7) for installing
-the Jupyter Notebook.
+While Jupyter runs code in many programming languages, **Python** 
+(Python 3.3 or greater, or Python 2.7) is a requirement for installing the Jupyter Notebook.
```

Comment:
Thank you for highlighting this outdated sentence. What about updating it and moving the version information further down?

```suggestion
is a requirement for installing the Jupyter Notebook. The Python version required
differs between Jupyter Notebook releases (e.g. Python 3.6+ for Notebook v6.3,
and Python 3.7+ for Notebook v7) .
```